bucket/tern-subtitle-file-translator.json

```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://tern.1c7.me",
+    "description": "Translate .srt .ass .vtt file",
+    "license": {
+        "identifier": "Shareware",
+        "url": "https://tern.1c7.me/pricing"
```

Comment:
Where is service pricing mentioned?

https://doc.tern.1c7.me/folder/pricing/#there-are-2-pricing

Reply:
So, do I need to replace the link with https://doc.tern.1c7.me/folder/pricing/ ?

Reply:
Overall you need both. One for general usage of application (commercial, company, personal, ...) and services for unlimited usage.

Maybe notes about pricing of services would be suitable.

Reply:
OK, I will add a note.
I really admire your professionalism and dedication. 👍